roles/KubevirtTemplateValidator/templates/service.yaml.j2

```diff
@@ -58,7 +58,8 @@ spec:
       serviceAccountName: template-validator
       containers:
         - name: webhook
-          image: {{ ssp_registry | default("quay.io/fromani") }}/{{ validator_image }}:{{ version }}
+          # `version` comes from the CR, while `validator_version` is our computed default
+          image: {{ ssp_registry | default("quay.io/fromani") }}/{{ validator_image }}:{{ (version == "automatic") | ternary(validator_version, version) }}
```

Comment:
We should handle the case where no version is provided by the CR or make the version field mandatory in the CRD, right?